copier/main.py

```diff
@@ -290,11 +304,19 @@ def render_folder(dst_path, rel_folder, flags):
         printf("create", display_path, style=STYLE_OK)
 
 
-def render_file(dst_path, rel_path, source_path, render, must_skip, flags):
+def render_file(
+    dst_path: Path,
+    rel_path: Path,
+    source_path: Path,
+    engine: Renderer,
+    must_skip: CheckPathFunc,
+    flags: Dict[str, bool],
+) -> None:
     """Process or copy a file of the skeleton.
     """
+    content: Optional[str]
     if source_path.suffix == ".tmpl":
-        content = render(source_path)
+        content = engine(source_path)
```

Comment:
I think `render` was a much better name. Without knowing what `engine` is, `engine(path)` is a mistery.
`render(path)` and `render.string(string)` were much clearer.

Reply:
Agreed `engine` is bad, because it is unclear.
There happened to be some naming collision confusing `mypy`.

As you have by now merged this **PR** I am going to open a separate **PR** to rename this identifier. Might take til next weekend for me to pull it off.